### ISSUE TITLE
 what is the alternative for get_game_variable(GameVariable variable) in VizDoom Version 1.1.5 and Version 1.1.4, Any help Please

### ISSUE THREAD

**Issue by adil25** [mwydmuch/ViZDoom#301]
Hello every one! what is the alternative for get_game_variable(GameVariable variable) in VizDoom Version 1.1.5 because i could not find this function, further it does not work in VizDoom Version 1.1.4 as well, Any help will be appreciated Please................

**Comment by mwydmuch** [comment]
Hi @adil25, 
so far we've only added methods to the API, `get_game_variable` is available from 1.0 and is still available in 1.1.5. This short example is definitely working:
```
game = DoomGame()
game.init()
game.get_game_variable(GameVariable.AMMO2)
```

**Comment by adil25** [comment]
Hi@mwydmuch,                           I installed both versions 1.1.4 and 1.1.5 but i failed to access 'get_game_variable (). in fact, it is not appearing in the IDE, please could you tell me what could be the reason? thanks
    On Thursday, April 12, 2018, 10:59:20 PM GMT+8, Marek Wydmuch <notifications@github.com> wrote:  
 
 
Hi @adil25,
so far we've only added methods to the API, get_game_variable is available from 1.0 and is still available in 1.1.5. This short example is definitely working:
game = DoomGame()
game.init()
game.get_game_variable(GameVariable.AMMO2)
